cmake/googletest/CMakeLists.txt

```diff
@@ -34,18 +34,18 @@ if(BUILD_TEST)
         set(ENABLE_STATIC_DEBUG "-Dgtest_force_shared_crt=ON")
```

Comment:
I guess you overlooked this in my other comment since you did not response. Doesn't `ENABLE_STATIC_DEBUG` create the same warning like the previously used `TOOLCHAIN_FILE`? It is used for all targets but only defined for `WIN32`.